docs/content/users/quickstart.md

```diff
@@ -359,29 +359,38 @@ The Laravel project type can be used for [Lumen](https://lumen.laravel.com/) lik
 
 === "Magento 2"
 
-    Normal details of a Composer build for Magento 2 are on the [Magento 2 site](https://devdocs.magento.com/guides/v2.4/install-gde/composer.html). You must have a public and private key to install from Magento’s repository. When prompted for “username” and “password” in `composer create`, it’s asking for your public and private keys.
+    Normal details of a Composer build for Magento 2 are on the [Magento 2 site](https://experienceleague.adobe.com/docs/commerce-operations/installation-guide/composer.html). You must have a public and private key to install from Magento’s repository. When prompted for “username” and “password” in `composer create`, it’s asking for your public key as "username" and private key as "password".
```

Comment:
Perhaps we can also add a tip here about credentials from https://ddev.readthedocs.io/en/stable/users/extend/in-container-configuration/ ?

```
mkdir -p ~/.ddev/homeadditions/.composer && ln -s ~/.composer/auth.json ~/.ddev/homeadditions/.composer/auth.json
```

I have set this up once and for all and I don't need to enter these credentials for new `magento2` projects.

Reply:
Reasonable to add. This is done automatically by this quickstart (putting in project homeadditions). 

Also, they provide an auth.json.example which could be used here.

Reply:
Added.